src/state/slices/opportunitiesSlice/resolvers/thorchainsavers/index.ts

```diff
@@ -155,6 +160,27 @@ export const thorchainSaversStakingOpportunitiesMetadataResolver = async ({
     }
   }
 
+  if (getConfig().REACT_APP_FEATURE_RUNEPOOL) {
+    stakingOpportunitiesById[thorchainAssetId as StakingId] = {
+      apy: undefined,
+      assetId: thorchainAssetId,
+      id: thorchainAssetId as StakingId,
+      provider: DefiProvider.ThorchainSavers,
+      // @TODO: calculate proper TVL
+      tvl: '10',
+      type: DefiType.Staking,
+      underlyingAssetId: thorchainAssetId,
+      underlyingAssetIds: [thorchainAssetId] as [AssetId],
```

Comment:
issue (blocking): underlying AssetIds for RUNEPool are [all PoL-enabled pools](https://gitlab.com/thorchain/thornode/-/blob/develop/docs/concepts/rune-pool.md#pol-enabled-pools), not only RUNE.
Note the "5 stablecoin pools" in the docs above isn't exactly true, as these two pools have 0 savers-depth so aren't (currently) effective as savers vaults, but this is already handled within our abstraction, just posting it here for context:
  - `BSC-USDC-0X8AC76A51CC950D9822D68B83FE1AD97B32CD580D`
  - `ETH-DAI-0X6B175474E89094C44DA98B954EEDEAC495271D0F`

```suggestion
       // TODO: we should probably fetch this from mimir with a polEnabledAssetIds query, but this works for now as all savers are PoL-enabled
      underlyingAssetIds: opportunityIds.filter(
        opportunityId => opportunityId !== thorchainAssetId,
      ) as unknown as AssetIdsTuple,
```

Note this currently won't look very nice, but is actually correct:

<img width="1088" alt="image" src="https://github.com/user-attachments/assets/1cb1fa92-4bef-40f8-a526-acf51f9ce320">
<img width="565" alt="image" src="https://github.com/user-attachments/assets/f41c6dd6-e882-4eac-80f8-f44337bcbb5c">

However, happy to keep that for a follow-up and add a TODO so we can do a three birds one stone one:

1. use the correct `underlyingAssetIds`
2. do the math to calculate current underlying ratios (from the looks of it, doesn't look like we can grab that info anywhere so we may have to represent it as an even split over all pools)
3. improve things at view-layer so this looks a lil' bit nicer

Reply:
Added a todo in [d387f46](https://github.com/shapeshift/web/pull/7430/commits/d387f46ef66e91fa6dc499cf57b03c1153695497), shouldn't be something blocking the work I guess